reset_docker.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# resets and removes the volumes from a docker container
+docker-compose down
+docker rm -f $(docker ps -a -q)
+docker volume rm $(docker volume ls -q)
```

Comment:
This seems like it would remove every docker container and volume, not just the ones related to this project. Can you think of a way to adjust it so that it only removes the DelifTLM instances? Aside from that very nice to have this in a script

Reply:
`docker-compose down -v` might be an option here: https://docs.docker.com/compose/reference/down/

Reply:
Thanks, that's a good one!